packages/sysutils/busybox/profile.d/98-busybox.conf

```diff
@@ -1,8 +1,9 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 export HOME="/storage"
-export PATH="/usr/bin:/usr/sbin"
+export PATH="/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
Isn't /bin already symbolic link to /usr/bin ? What is the purpose of having both ?